packages/vm/src/errors.rs

```diff
@@ -204,6 +211,66 @@ pub fn make_uninitialized_context_data<S: Into<String>>(kind: S) -> VmError {
     UninitializedContextData { kind: kind.into() }.build()
 }
 
+#[derive(Debug, Snafu)]
+pub enum FfiError {
+    #[snafu(display("Panic in FFI call"))]
+    ForeignPanic { backtrace: snafu::Backtrace },
+    #[snafu(display("bad argument passed to FFI"))]
+    BadArgument { backtrace: snafu::Backtrace },
+    #[snafu(display("Ran out of gas during FFI call"))]
+    OutOfGas,
```

Comment:
Any reason this does not get a backtrace? Looks inconsistent

Reply:
It just doesn't need it. When I'm done with things it will always be expected and handled, so that no one will observe the backtrace. On a higher level, it's not useful for users to see the backtrace even if we do format it and include it in the tx failure error message. It'll only contain information about the VM stack, not the contract stack, which users would care about.

Reply:
Okay, then let's just add the brackets for consistency as discussed here: https://github.com/CosmWasm/cosmwasm/pull/331#discussion_r426470873